packages/v4/patternfly-docs/content/get-started/design.md

```diff
@@ -5,49 +5,41 @@ section: get-started
 ---
 import './get-started.css';
 
-Whether you are converting from a PatternFly 3 application or starting fresh with the latest and greatest PatternFly design system, this document is intended to help provide designers with information regarding resources and support available.
+Whether you are converting from a preivous PatternFly application or starting fresh with the latest and greatest PatternFly design system, this page is intended to help provide designers with information regarding resources and support available.
```

Comment:
typo - should say "previous" instead of "preivous" haha

Reply:
Ooops good catch